src/plugins/gc-checklist/gc-checklist.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @title WET-BOEW Checklist plugin
+ * @overview Plugin contained to show an example of a custom WET checklist plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @delisma
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "gc-checklist",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+
+		if ( elm ) {
+			$elm = $( elm );
+
+			var $elms = $elm.find( $( "input[type='checkbox']" ) );
+
+			for ( var i = 0; i < $elms.length; i++ ) {
+				var domNode = $elms.get( i );
+				domNode.tabIndex = 0;
```

Comment:
Is the checkbox already focus-able element?

Reply:
@duboisp Back when they were `div` elements they were. But now since they piggy-back on the `input[type=radio]`, do they have the focus state by default?